packages/react-components/react-charts-preview/library/src/utilities/getWindow.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Verifies if an application can use DOM.
+ */
+export function canUseDOM(): boolean {
+  return (
+    // eslint-disable-next-line no-restricted-globals
+    typeof window !== 'undefined' &&
+    !!(
+      // eslint-disable-next-line no-restricted-globals, deprecation/deprecation
+      (window.document && window.document.createElement)
+    )
+  );
+}
```

Comment:
We can probably remove this file and use the already provided canUseDOM by v9 utilities and fluentprovider to get the window.

Reply:
Replaced the canUseDOM with the utilities equivalent.